pkg/minishift/addon/addon_meta.go

```diff
@@ -146,7 +147,7 @@ func requiredMetaTagsCheck(headers map[string]interface{}) error {
 func requiredOpenShiftVersionCheck(headers map[string]interface{}) error {
 	if headers[RequiredOpenShiftVersion] != nil {
 		if !checkVersionSemantic(headers[RequiredOpenShiftVersion].(string)) {
-			return errors.New("Add-on only support OpenShift version semantics eg. 3.6.0 or >3.6.0, <3.9.0 or >=3.5 etc.")
+			return errors.New("Add-on supports OpenShift version semantics (eg. assuming version 3.6.0, you would use 3.6.0, >3.6.0, <3.6.0 or >=3.6 etc.")
```

Comment:
Suggestion: `Add-on supports OpenShift version semantics (e.g. 3.6.0, >3.6.0, <3.6.0 or >=3.6 etc.)`. 

I do not think `assuming version 3.6.0, you would use` is technically correct because `>3.6.0` mean any version greater than v3.6.0.

Reply:
@LalatenduMohanty That makes sense. The `assuming version...` sounds like overkill now that I'm reading it back to myself a day later. I'll get this fixed up.

Reply:
@tylerauerbeck Can you please update this PR so we can merge it asap?